### DISCUSSION TITLE
GitHub Advanced Security Certification Course Winners 📣

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#140420] [Enterprise]
Congratulations to everyone who participated in our four week prep course for the [GitHub Advanced Security Certification](https://resources.github.com/learn/certifications/)! This was a wonderful month of learning and we are so thankful to see such a robust response from the community. We hope you feel fully prepared to ace your exam. 

### Certifications Voucher Winners 🏆

Thanks to the high level of participation, we are rewarding _10 participants_ with a **free GitHub Certifications exam voucher**! 

Meet our winners:

- @bos-hieu
- @davevad93
- @geopd
- @jccampanero
- @juanlujand
- @profile-palash
- @RishavKumarSinha
- @Salgado2004
- @suleman23194
- @Yordan-Ramchev

If you are a winner (one of the usernames listed above), to claim your voucher, fill out [this form](https://forms.gle/wHCRvbfvNRqkSrTeA) by **October 11, 2024 10am EST**. You can expect to receive your voucher by **October 18, 2024**. 

### What’s Next? ⏭️
* Sign up to take the [GHAS Certification Exam ](https://examregistration.github.com/login?ReturnUrl=%2Foverview)and get officially certified ✅
* Share your feedback in this discussion on the course materials and your experience taking the exam - both positive and negative are welcome 💬
* I’ll be updating some of the content to be more evergreen so new learners can prepare for the GHAS course at any time 🌲
* Stay tuned for more opportunities to prepare for certifications and more in the GitHub community 👀

We hope you enjoyed the course as much as we did! Let us know what you think below ⬇️

**Comment by davevad93** [comment]
Wow, thanks so much for the opportunity @queenofcorgis and good luck to everyone taking the certification exam! ❤️

**Comment by Salgado2004** [comment]
Thank you very much! ❤️ The prep course initiative was such an excellent idea! The contributions and connections were so cool and I got to meet other people with the same interest as me 😁 The materials were also very good, they helped me keep track of what I needed to focus on along with the study guides
I hope my breakdowns have also helped some you! Please share some feedback too! 🚀

Now I'm gonna wait for the exam and hope for the best, good luck for all of us! 😅 And I'll definitely come back to share the results

**Reply by queenofcorgis** [reply]
@Salgado2004 the breakdowns were great and delightfully unexpected addition to the course. As I update the materials in the discussion, I'll be linking to your additions as well 😄

**Reply by Salgado2004** [reply]
Hey guys, I'm back!
I know it took a while (the college finals were killing me 😵) but I finally took the exam for [GitHub Advanced Security](https://examregistration.github.com/certification/GHAS)! 
I'm so happy that I passed 😁 the preparation and shared contents were really useful! I'm gonna recommend to everyone I know!
Also, I'd like to thank you @queenofcorgis, for maintaining these discussions and keeping us motivated!

[Look my certification](https://www.linkedin.com/posts/leonardo-felipe-salgado-394312234_github-advanced-security-was-issued-by-github-activity-7268279662768762880-LuCA?utm_source=share&utm_medium=member_desktop)

**Reply by queenofcorgis** [reply]
Congratulations on passing @Salgado2004 !

**Comment by profile-palash** [comment]
Yup, big thanks to @queenofcorgis for keeping us in the learning loop with these awesome guides, motivations, resources and the vouchers to get certified! ❤️🙂

**Comment by juanlujand** [comment]
Great!! I already completed form!! I really appreciate this amazing journey
with you, I hope to share with you again in future

**Comment by jccampanero** [comment]
Sorry for the late reply.

Thank you very much, @queenofcorgis! It is a wonderful present!

As I previously mentioned, I genuinely appreciate the initiative. The course is full of great learning resources, and it has provided an excellent opportunity for sharing with others.

I wish the best for all my course mates. I've already reserved my voucher and will attempt the GHAS certification. I currently hold both the GitHub Foundations and GitHub Actions certifications. If you're interested, it could be an excellent opportunity to book some of these exams as well. Good luck with the exam!

Best regards!

**Comment by Xpollya** [comment]
I passed my **GitHub Advanced Security** successfully after putting in regular study time. **Pass4surexams** was useful for extra practice and helped me feel more prepared before taking the exam.
